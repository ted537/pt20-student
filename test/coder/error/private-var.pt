using output;

pkg package is 
    var x = 2;
end;

put(x:1);
putln;
