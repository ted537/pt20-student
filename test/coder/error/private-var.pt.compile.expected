semantic error, line 7: identifier not declared semantic error, line 7: expression operand required
